### PR TITLE
[Breaking] a11y: change login form

### PR DESCRIPTION
**Pull request by nzambello**
- Refactor login form: it had keyboard navigation issues with tabIndex
- Segment secondary text had not minimum contrast ratio, so I fixed it to textColor
- Login form is simpler and has better UX
- Add self registration config option

Screenshot_2021-04-22 Login

### CODE REVIEW

File: src/components/theme/Login/Login.jsx [hunk 1]
```diff
@@ -222,25 +229,7 @@ class Login extends Component {
...
-                                    id="registration form"
-                                    defaultMessage="registration form"
-                                  />
-                                </Link>
-                              ),
-                            }}
-                          />
-                        </p>
-                      </Grid.Column>
+                      <Grid.Column stretched width="12"></Grid.Column>
```
Comment (tiberiuichim) [inline src/components/theme/Login/Login.jsx:232]: I know this should be conditional on self-registration enabled. Do we have an endpoint for Plone settings?

Comment (sneridagh) [reply src/components/theme/Login/Login.jsx:232]: Not yet... it's part of that inglorious endpoint for "bootstrap" some things here and there, we could make it a static config for now.

**Review by sneridagh — COMMENTED** [review]
LGTM, but I would create that config for the register.

For the record, this is Quanta login form proposal, we are very close with this one :)

Screenshot 2021-04-25 at 10 56 20

Screenshot 2021-04-25 at 10 56 13

**Comment by sneridagh** [comment]
As we talked in the meeting, can we quickly implement a config setting that controls the "Register" toggle? eg. `config.settings.showSelfRegistration` ? Then tie it to the AnonTools link and the link in the login.

Thanks!

**Comment by nzambello** [comment]
@sneridagh added config option for self registration

**Comment by sneridagh** [comment]
Ok, let's merge this for Volto 13, ok? I think it's worth it.

**Review by tiberiuichim — APPROVED** [review]

**Pull request merged**